utils/iban.js

```diff
@@ -0,0 +1,51 @@
+// This code snippet checks the IBAN validity
+// The algorithm taken from:
+// https://en.wikipedia.org/wiki/International_Bank_Account_Number#Modulo_operation_on_IBAN
+
+const strip = iban => {
```

Comment:
this is on the verge of OCD, but `const strip = iban => iban.toUpperCase().replace(/\s+/g, '')` feels significantly more blissful

Reply:
The strip is quite needed, because the ban often comes in format 'CZXX XXXX XXXX XXXX', but the algorithm does not take the space there.

Reply:
I was just commenting on the way it's written, not on whether it's needed.

Reply:
ok we can close it then